conf/Makefile.arm-linux-toolchain

```diff
@@ -26,6 +26,8 @@ ifndef HARD_FLOAT
 endif
 endif
 
+PREFIX=/opt/arm-2012.03/bin/arm-none-linux-gnueabi
```

Comment:
Will break other linux builds

Reply:
fixed... will be addressed separately: see https://github.com/paparazzi/paparazzi/issues/2169